juni_farm/datagrabber/hcp_Aging_confounds_cat.py

```diff
@@ -0,0 +1,409 @@
+"""Provide a datagrabber for CAT-processed confounds in the HCP dataset."""
```

Comment:
Can you update the description?

Reply:
still needs an update i think @jkroell can you update the description here so it reflects the fact that this file is for HCP aging